packages/slice-machine/components/Simulator/components/Header/index.tsx

```diff
@@ -16,59 +10,32 @@ import { selectIsWaitingForIFrameCheck } from "@src/modules/simulator";
 import { selectSimulatorUrl } from "@src/modules/environment";
 import { isLoading } from "@src/modules/loading";
 import { LoadingKeysEnum } from "@src/modules/loading/types";
-import { ScreenDimensions } from "@lib/models/common/Screenshots";
 import { Button } from "@components/Button";
 import { AiFillCamera } from "react-icons/ai";
+import SliceMachineLogo from "@components/AppLayout/Navigation/Icons/SliceMachineLogo";
+import { ScreenDimensions } from "@lib/models/common/Screenshots";
 
 type PropTypes = {
   Model: ComponentUI;
   variation: Models.VariationSM;
-  handleScreenSizeChange: (screen: { size: Size }) => void;
-  size: Size;
-};
-
-const redirect = (
```

Comment:
⚠️  Unsure if we want to loose the link here on the slice name.
Feels like it could be useful.

Reply:
Not sure what you mean? This redirect was only used for changing the variation and was moved to the new Toolbar component